### PR TITLE
python3Packages.keyring: 23.4.0 -> 23.5.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by 0xbe7a — APPROVED** [review]
LGTM

//cc @SuperSandro2000

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 153187` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>bonfire</li>
<li>cq-editor</li>
<li>nagstamon</li>
<li>python38Packages.spyder</li>
<li>python38Packages.tvdb_api</li>
<li>python39Packages.tvdb_api</li>
<li>tvnamer</li>
<li>wapiti</li>
</ul>
</details>
<details>

  <summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>apache-airflow: <a href="https://termbin.com/gsmc9">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/a8b280fc02b7d30e0a181df43497c005/raw/index.html">pretty log </a></li>
<li>home-assistant: log was empty</li>
<li>python39Packages.jedi-language-server: log was empty</li>
  </ul>
</details>
<details>

  <summary>5 packages failed to build and are new build failures:</summary>
  <ul>
<li>(python39Packages.apache-airflow): log was empty</li>
<li>python38Packages.apache-airflow: log was empty</li>
<li>python38Packages.jedi-language-server: log was empty</li>
<li>python38Packages.pysychonaut: log was empty</li>
<li>python38Packages.requests-cache: log was empty</li>
  </ul>
</details>
<details>
<summary>
162 packages built:
</summary>
<ul>
<li>aws-google-auth</li>
<li>awslimitchecker</li>
<li>backintime (backintime-qt)</li>
<li>backintime-common</li>
<li>breezy (python39Packages.breezy)</li>
<li>cabal2nix</li>
<li>check_systemd</li>
<li>chia</li>
<li>cmake-language-server</li>
<li>coursera-dl</li>
<li>cryptop</li>
<li>cura</li>
<li>dep2nix</li>
<li>devpi-client</li>
<li>evdevremapkeys</li>
<li>flatpak-builder</li>
<li>gajim</li>
<li>haskellPackages.cabal2nix-unstable</li>
<li>hydra-unstable</li>
<li>i3pystatus</li>
<li>ioccheck</li>
<li>jira-cli</li>
<li>jrnl</li>
<li>keepkey_agent (python39Packages.keepkey_agent)</li>
<li>ledger-autosync</li>
<li>ledger_agent (python39Packages.ledger_agent)</li>
<li>lexicon</li>
<li>luigi</li>
<li>lutris</li>
<li>lutris-free</li>
<li>lutris-unwrapped</li>
<li>maestral (python39Packages.maestral)</li>
<li>maestral-gui</li>
<li>mirage-im</li>
<li>mnamer</li>
<li>mnemosyne</li>
<li>nix-prefetch-bzr</li>
<li>nix-prefetch-scripts</li>
<li>nix-update-source</li>
<li>numberstation</li>
<li>onlykey-agent</li>
<li>pantalaimon</li>
<li>pantalaimon-headless</li>
<li>pferd</li>
<li>pgcli (python39Packages.pgcli)</li>
<li>poetry2conda (python39Packages.poetry2conda)</li>
<li>protonvpn-gui</li>
<li>python38Packages.aioeafm</li>
<li>python38Packages.aiogithubapi</li>
<li>python38Packages.aiomodernforms</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.aria2p</li>
<li>python38Packages.astroquery</li>
<li>python38Packages.backoff</li>
<li>python38Packages.breezy</li>
<li>python38Packages.browser-cookie3</li>
<li>python38Packages.bugwarrior</li>
<li>python38Packages.censys</li>
<li>python38Packages.check-manifest</li>
<li>python38Packages.geocachingapi</li>
<li>python38Packages.gtts</li>
<li>python38Packages.hypothesis-auto</li>
<li>python38Packages.jira</li>
<li>python38Packages.karton-mwdb-reporter</li>
<li>python38Packages.keepkey_agent</li>
<li>python38Packages.keyring</li>
<li>python38Packages.keyrings-cryptfile</li>
<li>python38Packages.launchpadlib</li>
<li>python38Packages.ledger_agent</li>
<li>python38Packages.libagent</li>
<li>python38Packages.maestral</li>
<li>python38Packages.mwdblib</li>
<li>python38Packages.myfitnesspal</li>
<li>python38Packages.nagiosplugin</li>
<li>python38Packages.ndjson</li>
<li>python38Packages.ofxclient</li>
<li>python38Packages.pgcli</li>
<li>python38Packages.plyer</li>
<li>python38Packages.poetry</li>
<li>python38Packages.poetry2conda</li>
<li>python38Packages.protonvpn-nm-lib</li>
<li>python38Packages.pyairnow</li>
<li>python38Packages.pyicloud</li>
<li>python38Packages.pyiqvia</li>
<li>python38Packages.pyopenuv</li>
<li>python38Packages.pypiserver</li>
<li>python38Packages.pyprosegur</li>
<li>python38Packages.pysnow</li>
<li>python38Packages.python-daemon</li>
<li>python38Packages.python-miio</li>
<li>python38Packages.salmon-mail</li>
<li>python38Packages.simplisafe-python</li>
<li>python38Packages.tesla-wall-connector</li>
<li>python38Packages.teslajsonpy</li>
<li>python38Packages.toggl-cli</li>
<li>python38Packages.toonapi</li>
<li>python38Packages.trezor_agent</li>
<li>python38Packages.twine</li>
<li>python38Packages.url-normalize</li>
<li>python38Packages.velbus-aio</li>
<li>python38Packages.versionfinder</li>
<li>python38Packages.wled</li>
<li>python38Packages.yalesmartalarmclient</li>
<li>python39Packages.aioeafm</li>
<li>python39Packages.aiogithubapi</li>
<li>python39Packages.aiomodernforms</li>
<li>python39Packages.ansible-kernel</li>
<li>python39Packages.ansible-runner</li>
<li>python39Packages.aria2p</li>
<li>python39Packages.astroquery</li>
<li>python39Packages.backoff</li>
<li>python39Packages.browser-cookie3</li>
<li>python39Packages.bugwarrior</li>
<li>python39Packages.censys</li>
<li>python39Packages.check-manifest</li>
<li>python39Packages.geocachingapi</li>
<li>python39Packages.gtts</li>
<li>python39Packages.hypothesis-auto</li>
<li>python39Packages.jira</li>
<li>python39Packages.karton-mwdb-reporter</li>
<li>python39Packages.keyring</li>
<li>python39Packages.keyrings-cryptfile</li>
<li>python39Packages.launchpadlib</li>
<li>python39Packages.libagent</li>
<li>python39Packages.mwdblib</li>
<li>python39Packages.myfitnesspal</li>
<li>python39Packages.nagiosplugin</li>
<li>python39Packages.ndjson</li>
<li>python39Packages.ofxclient</li>
<li>python39Packages.plyer</li>
<li>python39Packages.poetry</li>
<li>python39Packages.protonvpn-nm-lib</li>
<li>python39Packages.pyairnow</li>
<li>python39Packages.pyicloud</li>
<li>python39Packages.pyiqvia</li>
<li>python39Packages.pyopenuv</li>
<li>python39Packages.pypiserver</li>
<li>python39Packages.pyprosegur</li>
<li>python39Packages.pysnow</li>
<li>python39Packages.pysychonaut</li>
<li>python39Packages.python-daemon</li>
<li>python39Packages.python-miio</li>
<li>python39Packages.requests-cache</li>
<li>python39Packages.salmon-mail</li>
<li>python39Packages.simplisafe-python</li>
<li>python39Packages.tesla-wall-connector</li>
<li>python39Packages.teslajsonpy</li>
<li>python39Packages.toggl-cli</li>
<li>python39Packages.toonapi</li>
<li>trezor_agent (python39Packages.trezor_agent)</li>
<li>twine (python39Packages.twine)</li>
<li>python39Packages.url-normalize</li>
<li>python39Packages.velbus-aio</li>
<li>python39Packages.versionfinder</li>
<li>python39Packages.wled</li>
<li>python39Packages.yalesmartalarmclient</li>
<li>sublime-music</li>
<li>theharvester</li>
<li>urlwatch</li>
<li>vcstool</li>
<li>wlc</li>
</ul>
</details>
